### PR TITLE
Update Get-Team.md

### PR DESCRIPTION
**Pull request by rajgodwin**
Updated the old note which was no longer applicable with a new note.

### CODE REVIEW

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by tseward** [comment]
I would suggest a small verbiage change; instead of using "At times", use "May".

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/Get-Team.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Get-Team [-User <String>] [-Archived <Boolean>] [-Visibility <String>] [-Display
 This cmdlet supports retrieving teams with particular properties/information, including all teams that a specific user belongs to, all teams that have been archived, all teams with a specific display name, or all teams in the organization.
 
 >[!NOTE]
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-Team.md:34]:
```suggestion
> [!NOTE]
```

File: teams/teams-ps/teams/Get-Team.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Get-Team [-User <String>] [-Archived <Boolean>] [-Visibility <String>] [-Display
...
->Depending on the number of teams and O365 Groups in your organization and which filters you are using, this cmdlet can take upwards of ten minutes to run.  Some of the input parameters are guaranteed unique (e.g. GroupId), and others serve as filters (e.g. -Archived).
+>At times, Get-Team returns multiple results matching the input and not just the exact match for attributes like DisplayName/MailNickName. This is a known behaviour.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-Team.md:35]:
```suggestion
> Get-Team may return multiple results matching the input and not just the exact match for attributes like DisplayName/MailNickName. This is known behavior.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor changes, including @tseward's suggestion.

**Pull request merged**